cfme/infrastructure/virtual_machines.py

```diff
@@ -939,14 +939,14 @@ def reconfigure(self, new_configuration=None, changes=None, cancel=False):
         if new_configuration:
             changes = self.configuration.get_changes_to_fill(new_configuration)
 
-        any_changes = any(v not in [None, []] for v in changes.values())
+        any_changes = any(v not in [None, []] for v in list(changes.values()))
```

Comment:
actually `any(changes.values())` will probably do

Reply:
will take a look - result of 2to3 `dict` checker.

Reply:
I don't think its safe to go as far as `any(changes.values())` here, because its building/comparing configuration dicts for form filling.

There's a distinct difference between `None` and boolean `False` here, as `False` actually indicates a changed form value, where as `None` or an empty list indicates no change to the form for that key.